tests/sse-mockserver/src/main/kotlin/com/apollographql/apollo/sample/server/routing/SseSideChannelRouter.kt

```diff
@@ -0,0 +1,53 @@
+package com.apollographql.apollo.sample.server.routing
+
+import com.apollographql.apollo.sample.server.sse.SseSideChannelInteractor
+import io.ktor.application.call
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.request.receive
+import io.ktor.response.respondText
+import io.ktor.routing.Routing
+import io.ktor.routing.post
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class SseSideChannelRouter : Router() {
+  companion object {
+    const val SIDE_CHANNEL_PATH = "v1/subscription"
+  }
+
+  private val interactor = SseSideChannelInteractor()
+
+  override fun routing(routing: Routing) {
+    routing.post("/$SIDE_CHANNEL_PATH") {
+
+      withContext(Dispatchers.IO) {
+        try {
+          call.receive<SseTransportMessage.ClientRequest>()
+              .let { interactor.processRequest(it) }
+              .let { serialize(it) }
+              .let {
+                call.respondText(
+                    text = it,
+                    contentType = ContentType.Application.Json,
+                    status = HttpStatusCode.Created,
+                )
+              }
+
+
+        } catch (t: Throwable) {
+          t.printStackTrace()
```

Comment:
Nit: I'd add a TODO here to not forget handling properly it later 😊

Reply:
Certainly needs attention later. 
I'm scheduling error handling code for very late, but keen to have conversations earlier. In this situation, is there any special recommendation for say, malformed request?

I'm also not planning to be very strict on the server implementation since its main role is support the dev and test of the client, but still worth following the basics.